katdal/test/test_applycal.py

```diff
@@ -153,24 +153,50 @@ class TestComplexInterp(object):
     """Test the :func:`~katdal.applycal.complex_interp` function."""
     def setup(self):
         self.xi = np.arange(1., 10.)
-        self.yi = np.exp(2j * np.pi * self.xi / 10.)
+        self.yi_unit = np.exp(2j * np.pi * self.xi / 10.)
         rs = np.random.RandomState(1234)
         self.x = 10 * rs.rand(100)
```

Comment:
This probably needs to be sorted to guarantee that interpolation is defined. You've done that for test_left_right but not the other tests.

Reply:
It's `xi` that needs to be sorted, not `x`... I sorted `x` in `test_left_right` so that I could easily find the left-most and right-most extrapolations in order to check them.

Reply:
Ah, I get it now.